lib/pf/pfcmd/checkup.pm

```diff
@@ -1282,6 +1283,42 @@ sub portal_modules {
     }
 }
 
+=head2 cluster
+
+Validate the configuration of the cluster
+
+=cut
+
+sub cluster {
+    require pf::cluster;
+    require pf::ConfigStore::Interface;
+    require pfconfig::namespaces::config::Cluster;
+
+    my $int_cs = pf::ConfigStore::Interface->new;
+    my @ints = @{$int_cs->readAll('name')};
+    my @servers = @pf::cluster::cluster_servers;
+
+    my $cluster_config = pfconfig::namespaces::config::Cluster->new;
+    $cluster_config->build();
+    
+    unshift @servers, $cluster_config->{_CLUSTER};
+
+    # Check each member configuration
+    foreach my $server (@servers){
+        my $server_name = $server->{host};
+        unless(defined($server->{management_ip}) && valid_ip($server->{management_ip})){
+            add_problem($FATAL, "management_ip is not defined for $server_name");
```

Comment:
IMO we should split this into to checks two checks.
One for an undefined management_ip.
The other for an invalid ip address.

Since those will require two different actions.